forage-android/src/main/java/com/joinforage/forage/android/network/model/ForageApiResponse.kt

```diff
@@ -6,9 +6,23 @@ internal val UnknownErrorApiResponse = ForageApiResponse.Failure.fromError(
     ForageError(500, "unknown_server_error", "Unknown Server Error")
 )
 
+/**
+ * A model that represents the possible types of responses from the Forage API.
+ *
+ */
 sealed class ForageApiResponse<out T> {
+    /**
+     * A model that represents a success response from the API.
+     *
+     */
     data class Success<out T>(val data: T) : ForageApiResponse<T>()
 
+    /**
+     * A model that represents a failure response from the API.
+     *
+     * @property errors A list of [ForageError] instances that you can unpack to programmatically
+     * handle the error and display the appropriate customer-facing message.
+     */
     data class Failure(val errors: List<ForageError>) : ForageApiResponse<Nothing>() {
```

Comment:
Can we link to relevant Forage docs that talk about displaying customer-facing error copy?

Reply:
Hmmm this is one I want to think about! That particular doc is specific to BRDs, so I am worried it's a little confusing. I can link for now, but need to think on this more longterm as well.